airflow/config_templates/config.yml

```
@@ -1264,6 +1264,14 @@ hive:
 webserver:
   description: ~
   options:
+    config_file:
+      description: |
+        Path of webserver_config.py file used for configuring the webserver parameters with
+        respect to the AIRFLOW_HOME
```

```suggestion
        Path of webserver_config.py file used for configuring the webserver parameters
```

It can really be anywhere, so you can drop that part of it.

---

We might even consider not having `webserver_config.py` in the description either, since it really could be named anything now!

---

Oh yes, great suggestion. Made the changes 👍🏽